src/app/api/auth/[...nextauth]/async/getJsonData.ts

```diff
@@ -0,0 +1,22 @@
+import { getJsonData } from './fetchService'; 
+
+interface UserData {
+    id: number;
+    name: string;
+    email: string;
+    // ... other user fields
+}
+
+async function fetchUserData() {
+    try {
+        const url = 'https://example.com/api/users'; 
```

Comment:
Nice but this will be better if you take the data from the env file in this case you can use something like
`process.env.REACT_APP_BASE_URL`
and their use should be
```
const url = `${process.env.REACT_APP_BASE_URL}/api/users`
```
 please fix it